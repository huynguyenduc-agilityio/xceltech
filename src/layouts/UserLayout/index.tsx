import { lazy } from 'react';
import { Outlet } from 'react-router-dom';

// Components
import { Navbar } from '@/components';

const ConfirmDialog = lazy(() => import('@/components/ConfirmDialog'));

const UserLayout = () => {
  return (
    <>
      <Navbar />
      <main className="min-h-screen p-14 bg-blue-light">
        <Outlet />
      </main>
      <ConfirmDialog />
    </>
  );
};

export default UserLayout;
