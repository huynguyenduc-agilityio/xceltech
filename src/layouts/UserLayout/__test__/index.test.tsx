import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

// Layout
import UserLayout from '..';

describe('UserLayout', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <UserLayout />
      </MemoryRouter>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
