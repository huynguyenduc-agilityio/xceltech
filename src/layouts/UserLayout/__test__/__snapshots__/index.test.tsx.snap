// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserLayout should match snapshot 1`] = `<DocumentFragment />`;
